docs/tech-reference/iso-plugins.rst

```diff
@@ -152,3 +152,6 @@ configuration parameters are required.
  enabled, the distributed repository will become a protected repository. The given CA certificate will be used
  to verify the clients' entitlement certificates. If this certificate is not provided, the repository will be an
  unprotected repository.
+
+``relative_url``
+Relative path at which the repository will be served.
```

Comment:
What is this path relative to? How about writing something like:

> The path at which this repository will be served, relative to the the ISO distributor's ``root``.

Except that the ISO distributor has no ``root``option, so I don't know what this option is supposed to do.